src/world.cpp

```diff
@@ -127,8 +117,12 @@ World::World(Json::Value materials_json, Json::Value biome_data, unsigned int x_
 
     std::cout << "start of terrain\n";
 
+    constexpr int macro_tile_size = 32;
+    constexpr int height = 128;
+    constexpr int seed = 5;
```

Comment:
Put it in a header file the whole point is NOT to duplicate it

Reply:
I have no idea what you wnat me to do. This runs one (1) time there is no proformance improvement.